platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/src/test/java/com/swirlds/platform/test/consensus/ConsensusTests.java

```diff
@@ -42,6 +47,30 @@ public static void initConfig() {
         new TestConfigBuilder().getOrCreateConfig();
     }
 
+    /**
+     * Injects a platform context with the birth round ancient threshold set to the value in the test parameters and
+     * adds the temp directory for writing marker files.
```

Comment:
This method doesn't appear to be doing anything with marker files, I assume this comment is stale?